Src/GBX.NET/Engines/Game/CGameCtnCollection.chunkl

```diff
@@ -67,28 +67,28 @@ CGameCtnCollection 0x03033000
  ident Vehicle
 
 0x00B
- int
+ int BlocksShadow
 
 0x00C
  bool IsEditable
- int
+ int<ECollectionType> CollectionType
 
-0x00D
+0x00D //IconFid(0xc0), IconSmallFid(0xc4)
```

Comment:
i'd ignore the hex offsets really, they dont help much in the project
just keep it as `IconFid, IconSmallFid`

also, prefer commenting with a space like so: `// my comment` instead of `//my comment`